src/test/java/it/TestApplication.java

```diff
@@ -17,7 +17,7 @@
 
 import org.junit.Test;
 
-public class TestApplication extends EndpointTest {/*
+public class TestApplication extends EndpointIT {/*
```

Comment:
Rename this file to be ApplicationIT.java.  Otherwise, surefire will pick it up.